index.js

```diff
@@ -0,0 +1,81 @@
+const express = require('express');
+const bcrypt = require('bcryptjs');
+const helmet = require('helmet');
+const cors = require('cors');
+
+const db = require('./data/dbConfig.js');
+const Users = require('./users/users-model.js');
```

Comment:
It might be more appropriate to call this `Users` variable something like `userModel` or `userHelpers` for the sake of clarity and readability.